### PR TITLE
Subscribe to my blog!

### PR DESCRIPTION
**Pull request by kartik-madhak**
## What

This PR adds ability to subscribe to blogs - basically a small section below the share buttons (See the screenshot below).
It uses a service called [tinyletter](https://tinyletter.com/) to manage subscriptions, since it is free of cost, easier to setup and also has email verification. 
- [ ] We might want to change the subscribe message (see screenshots below)
- [ ] We might want to signup to tinyletter using a company email address and configure the subscribe form there. 

## How

- Adds a new partial for the newsletter
- Use the html template provided in the tinyletter website inside the partial and add styling to it

## Why

Why were these changes made?

- To track visitors who like our blogs

## Screenshots (If applicable)

Dark theme: -
image

Light theme: -
image

Smaller devices: -
image

# Checklist

- [x] My code follows the style guidelines of this project and is properly formatted
- [x] I have performed a self-review of my own changes
- [x] All existing tests pass, and I have added new ones if required

### CODE REVIEW

**Review by 2KAbhishek — APPROVED** [review]

**Comment by sapanparikh18** [comment]
I love this, even the language is so cool!
Was it intentional to keep the word "Monitor" in the second statement?

Also, we should try it in different positions, the last thing on the page is not so inviting! People may never subscribe!

**Comment by sapanparikh18** [comment]
We also need to do one more PR to bring this registration box to blog.incubyte.co landing page

**Review by sapanparikh18 — APPROVED** [review]

**Comment by kartik-madhak** [comment]
> I love this, even the language is so cool! Was it intentional to keep the word "Monitor" in the second statement?
> 
> Also, we should try it in different positions, the last thing on the page is not so inviting! People may never subscribe!

Glad to hear that, personally thought it would be a little too sassy to show that message!
The word "monitor" in this context means that we will monitor the viewer's scroll speed next time they visit our blog! XD

Also about the position, I do feel like it is not getting much attention at the bottom, we tried another method by popping up a small dialog box from the bottom of the page when user's scroll reaches a certain value but didn't go with that because it might be disruptive to user's experience... 
Alternatively, we could also just move it above the Share buttons section and add some catchy image [like this](https://www.google.com/url?sa=i&url=https%3A%2F%2Fwww.pinterest.com%2Fpin%2F719309371710900325%2F&psig=AOvVaw2cejDyJ38t_iTToMaBhwdH&ust=1674744329841000&source=images&cd=vfe&ved=0CBAQjRxqFwoTCNDu_vj64vwCFQAAAAAdAAAAABAH)

File: layouts/partials/subscribe.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<div class="mt-5 newsletter">
+  <form
+    action="https://tinyletter.com/incubyte-test"
+    method="post" target="popupwindow"
+    onsubmit="window.open('https://tinyletter.com/incubyte-test', 'popupwindow', 'scrollbars=yes,width=800,height=600');return true">
```
Comment (akshayvadher) [inline layouts/partials/subscribe.html:5]: incubyte instead of incubyte-test?

**Pull request merged**